Dockerfile

```diff
@@ -21,24 +21,26 @@ RUN git clone https://github.com/robshakir/pyangbind /workspace/pyangbind-repo
 
 RUN apt-get update
 RUN apt install -y python3-pip
-RUN pip3 install virtualenv
+RUN apt install -y virtualenv
 #RUN virtualenv /workspace/pyangvenv
 
 # Install packages for latest pyang venv.
-RUN pip3 install wheel && \
-        pip3 install pyaml && \
-        pip3 install enum34 && \
-        pip3 install jinja2 && \
-        pip3 install setuptools && \
-        pip3 install --no-cache-dir -r /workspace/oc-pyang-repo/requirements.txt && \
-        pip3 install --no-cache-dir -r /workspace/pyangbind-repo/requirements.txt
-#        pip3 install pyangbind
+RUN apt install python3-wheel && \
+        pip3 install pyaml enum34 --break-system-packages && \
+        apt install -y python3-jinja2 && \
+        apt install -y python3-setuptools && \
```

Comment:
i usually find pip to work better when run inside a virtualenv.  Installing virtualenv via apt is ok.  But once done, should the virtualenv be activated (~/virtualenv/bin/activate) and then the pip3 installs done?

Reply:
I didn't do this because the validators already use virtualenv (https://github.com/openconfig/models-ci/blob/master/validators/pyang/test.sh), so I would be forcing them to nest virtualenvs, which is not well supported. The right fix is to move these all to the validators but that would be a refactoring effort and we haven't seen a strong need for that yet.

I'll add a comment documenting this.